kubernetes/helm_charts/logging/kibana/templates/deployment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "fullname" . }}
+  name: kibana
```

Comment:
why are we changing public helm charts? it'll be difficult to upgrade, right. can't we override this using `-f values.yaml`

Reply:
you can have a look in sunbird-monitoring role in k8s

Reply:
We will not use the public helm chart for this. We will keep this internal only as these changes are very rare. When we do, we have a bigger worry to do ES first than Kibana,